barretenberg/cpp/pil/vm2/trees/merkle_check.pil

```diff
@@ -161,17 +161,14 @@ namespace merkle_check;
     pol commit read_output_hash;
     pol commit write_output_hash;
 
-    pol commit constant_2;
-    sel * (constant_2 - 2) = 0;
-
```

Comment:
We can just use `start` and `end` for the lookups in this file since they are both single row hashes, meaning we don't need this column